.github/workflows/smart_asa_ci.yaml

```diff
@@ -29,10 +29,14 @@ jobs:
 
     - name: Run Algorand Sandbox
       run: |
-        docker run -d -p 4001:4001 -p 4002:4002 matteojug/algorand-sandbox-dev:3.7.2
+        docker run -d -p 4001:4001 -p 4002:4002 matteojug/algorand-sandbox-dev:3.9.2
 
     - name: Sleep to allow the sandbox to start
       run: sleep 10
 
     - name: Run pytest
       run: pipenv run pytest
+
+    - name: Stop running images
+      run: |
+        docker stop $(docker ps -q --filter ancestor=matteojug/algorand-sandbox-dev:3.9.2)
```

Comment:
Is this required?

Reply:
Not strictly required. Just making the CI homogeneous with other repos.